scripts/ci/run-jenkins.sh

```diff
@@ -355,7 +355,7 @@ if [[ ${CI_TAGS} == *'webassembly'* ]] || [[ ${CI_TAGS} == *'wasm'* ]];
             ${TESTCMD} --label=system-core --timeout=60m $gnumake -C sdks/wasm run-all-System.Core
             for suite in ${xunit_test_suites}; do ${TESTCMD} --label=xunit-${suite} --timeout=30m $gnumake -C sdks/wasm run-${suite}-xunit; done
             # disable for now until https://github.com/mono/mono/pull/13622 goes in
-            ${TESTCMD} --label=debugger --timeout=20m $gnumake -C sdks/wasm run-debugger-tests
+            ${TESTCMD} --label=debugger --timeout=60m $gnumake -C sdks/wasm run-debugger-tests
```

Comment:
If a test hangs this is a lot longer.  Should we just double it?

Also It might make sense to move the debugger tests before the system tests since the debugger is more likely to fail at this point.